lib/components/base-components/NormalComponent/NormalComponent_doInitialPcbFootprintStringRender.ts

```diff
@@ -154,4 +154,54 @@ export function NormalComponent_doInitialPcbFootprintStringRender(
   ) {
     component.add(footprint as Footprint)
   }
+
+  if (
+    typeof footprint === "object" &&
```

Comment:
since circuit json is always an array, you could do an array check here (idk why you add objects directly, seems weird)